.github/workflows/build_and_deploy.yml

```
@@ -0,0 +1,65 @@
+          push: true
+          tags: ${{ env.IMAGE_NAME }}:${{ env.IMAGE_TAG }},${{ env.IMAGE_NAME }}:latest
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
+
+  deploy-dev:
+    name: Deploy from ${{ github.ref_name }}/${{ github.sha }}
+    runs-on: ubuntu-latest
+    needs: build-and-push
+    steps:
+      - name: Trigger dev deployment
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:11:8: Double quote to prevent globbing and word splitting

(shellcheck)